### PR TITLE
Qualys VM v2 data connector, workbook , detection

### PR DESCRIPTION
**Pull request by v-maudan**
Fixes #

Original PR raised by pbankar28:[ Added pagination support and flatten the data. by pbankar28 · Pull Request #969 · Azure/Azure-Sentinel (github.com) ](https://github.com/Azure/Azure-Sentinel/pull/969)

- Updated data connector and workbook, detection queries as per new changes.
- Updated custom table schema to pass KQL validations.

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by v-maudan** [comment]
@pbankar28 , As per latest Qualys V2 changes created new data connector, workbook and detections without impacting existing one. 
Can you please test and review this PR. let me know if any changes required. Thanks!

File: DataConnectors/Qualys VM V2/QualysVM_V2_API_FunctionApp.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        {
+            "metricName": "Total data received",
+            "legend": "QualysHostDetection_CL",
+            "baseQuery": "QualysHostDetection_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Vulerabilities detected",
+            "query": "QualysHostDetection_CL\n | extend Vulnerability = tostring(Results_0_s)\n | summarize count() by Vulnerability\n | top 10 by count_"
```
Comment (pbankar28) [inline DataConnectors/Qualys VM V2/QualysVM_V2_API_FunctionApp.json:16]: You may replace Results_0_s to QID_s

Comment (v-maudan) [reply DataConnectors/Qualys VM V2/QualysVM_V2_API_FunctionApp.json]: @pbankar, Done. Updated detection query as well: Please review and let me know if any other changes required. Thanks!.

**Comment by v-maudan** [comment]
@pbankar28 , Can you please review and approve this PR ? Thanks!

**Comment by v-rucdu** [comment]
@pbankar28 : Could you please review the changes and provide sign-off? 
This is blocking us from check-in... Thanks!!!

**Review by v-rucdu — APPROVED** [review]

**Pull request merged**